### PR TITLE
Fix for non-medium domain posts (for Firefox)

### PR DESCRIPTION
**Pull request by j3soon**
- Requires user to enable the plugin in Incognito Mode in firefox.
- Can work in non-medium domain.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 Try Medium with membership-features!
 
+**If you are using Firefox, must allow this extension in incognito mode!**
```
Comment (swapagarwal) [inline README.md]: HI Johnson, can you shed more light on this?

Comment (j3soon) [reply README.md]: The line below requires the extension to be allowed in incognito mode. (Firefox have a option now to specify whether the extension is allowed in incognito mode)
```
browser.windows.create({"url": url, "incognito": true});
```

Comment (j3soon) [reply README.md]: After installing the extension, set `Tools > Add-ons > Mediumship > Run in Private Windows` to `Allow`

image

Comment (swapagarwal) [reply README.md]: Sorry I didn't understand. 😅
In the normal mode, clicking on this extension opens the tab in incognito mode. Why does the extension be allowed in incognito mode?

Comment (j3soon) [reply README.md]: Oh, I think I didn't express it clear enough.
Seems like `opening a window in incognito mode` requires the extension to be allowed in incognito mode in firefox.

Comment (swapagarwal) [reply README.md]: Got it! This behavior is weird though.

Comment (j3soon) [reply README.md]: I got quite confused at first too. I found the error message in the extension debugging console and solved it by allowing the extension in incognito mode. I'm wondering if there are other ways to open an incognito tab without the permission.

Comment (swapagarwal) [reply README.md]: So I got to rewrite the whole thing from scratch, and we have v2.0 now! It's being published in Chrome Webstore right now. Once we verify that everything works as expected, maybe we can do the same for the Firefox version.
P.S. The new version works wholly in background, without any user-intervention! 😉

Comment (swapagarwal) [reply README.md]: 2.0 looks good to me. Can you check it out at your end? Let's bump up the firefox version too! 👊

Comment (swapagarwal) [reply README.md]: We can post to Product Hunt at midnight (13 hours from now): https://whattimeisitatproducthunt.com/

Comment (swapagarwal) [reply README.md]: Checking in if you can make it within the next 4 hours... 😅

Comment (j3soon) [reply README.md]: I've just finished reading the code. I think maybe we should keep the user-clicking action on any other domain name?

Comment (j3soon) [reply README.md]: By that way we can fix #6, if the domain name isn't medium.com

Comment (swapagarwal) [reply README.md]: Yes, makes sense! 👍
https://github.com/swapagarwal/mediumship/issues/6#issuecomment-500234833

Comment (j3soon) [reply README.md]: OK. I'll try to update it in the next 2 hours.

Comment (swapagarwal) [reply README.md]: Awesome! Send me your PH username if you'd like to be added as a maker [here](https://www.producthunt.com/posts/mediumship).

Comment (j3soon) [reply README.md]: My PH username is `j3soon`. By the way, what's this url `https://t.co/JV5396gd2O` pointing to?

Comment (swapagarwal) [reply README.md]: It points to [medium.com](http://www.getlinkinfo.com/info?link=https://t.co/JV5396gd2O) 😏
(Added on [PH](https://www.producthunt.com/posts/mediumship))

Comment (j3soon) [reply README.md]: But I think we're using twitter as the referrer as mentioned in #5 ?

Comment (swapagarwal) [reply README.md]: So that means using any Twitter URL as the referrer, e.g. `https://t.co/JV5396gd2O` and not Twitter itself. 😅

Comment (j3soon) [reply README.md]: Got it! I'll close this PR and open another one.

**Pull request closed without merging**